.github/workflows/package-ci.yml

```diff
@@ -345,7 +345,7 @@ jobs:
       - name: Test open app
         run: |
           set -eux
-          tar -xzvf dist/parsec-${{ env.WHEEL_VERSION }}-macos-amd64.tar.bz2
```

Comment:
I don't see a spelling mistake here. Is it a false positive or just a style we should respect

Reply:
The change is indeed because of a false positive and I wasn't felling it to add to the list of ignored word

Reply:
I've disable `cspell` for this specific line and revert the change